novm/manager.py

```diff
@@ -227,7 +227,7 @@ def state(output):
                 index=1+len(nics)+len(disks),
                 pci=not(nopci),
                 tag="root",
-                tempdir=self._instances.file(str(os.getpid())),
+                tempdir=self._myInstance.file("writable"),
```

Comment:
Sorry, I may have been ambiguous.

The self._myInstance stuff is a no-go. The NovmManager() should be useable within other scripts, and may be used to create and manage multiple "containers" simultaneously.

You could make the instance a parameter to the state() closure, and just a regular variable below.